haystack/document_stores/deepsetcloud.py

```diff
@@ -17,7 +17,7 @@ def __init__(
         self,
         api_key: str = None,
         workspace: str = "default",
-        index: str = "default",
+        index: Optional[str] = None,
```

Comment:
Let's add a short description of the create-pipeline-use-case to init. Something like that:

You can also use this DocumentStore to create new pipelines on deepset Cloud. To do that take the following steps:
- create a new DeepsetCloudDocumentStore without an index (e.g. `DeepsetCloudDocumentStore()`)
- create query and index pipelines using this DocumentStore
- call `Pipeline.save_to_deepset_cloud()` passing the pipelines and a `pipeline_config_name`
- call `Pipeline.deploy_on_deepset_cloud()` passing the `pipeline_config_name`

Reply:
Done.